content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Troubleshooting CNI plugin-related errors
+content_type: task
+reviewers:
+- mikebrow
+weight: 10
+---
+
+<!-- overview -->
+
+To avoid CNI plugin-related errors, verify that you are using or upgrading to a
+container runtime that has been tested to work correctly with your version of
+Kubernetes.
+
+For example, the following container runtimes have been tested with Kubernetes
+v1.24:
+
+* containerd v1.6.4 and later, v1.5.11 and later
```

Comment:
... the network fixes needed have been merged into the containerd [1.6.x branch](https://github.com/containerd/containerd/commits/release/1.6), we held a community meeting Thursday (yesterday) morning and all agreed a service release (1.6.4) is needed and will be provided within days if not today early next week .. testing of what that release will look like can be done against [this commit](https://github.com/containerd/containerd/commit/6eeeaaddca18b0d8c3676e706e95680d01caafee) there may also be one or two more unrelated commits coming in one for a [k8s volume mount relabel issue](https://github.com/containerd/containerd/commit/6eeeaaddca18b0d8c3676e706e95680d01caafee) .. We are just about out of time, but we will do one more release primarily for k8s fixes.

Reply:
@dims will chat it up to decide if you want to run on a merge commit or wait for the service release tag

Reply:
Correct, I understand that Mike. However, my concern is writing it out before actually testing it would not be right by our audience especially in the context of the release blog since that goes out on the day of the release i.e. May 3rd. I'm just considering rephrasing it to paint a more accurate picture for the release blog. This is a troubleshooting doc and as such, is not a release blocker.

Reply:
Sure, thanks @mikebrow 
FYI : @JamesLaverack , @nate-double-u , @chris-short , @mickeyboxell, @reylejano @natalisucks

Reply:
@divya-mohan0209 rephrase suggested above.. thx!!

Reply:
Thank you so much!

Reply:
@rolfedh - please could you make the changes as suggested by @mikebrow ? Thank you so much :)

Reply:
@mikebrow  What is the suggestion for line 19?

Reply:
The status of this comment is "Resolved." @divya-mohan0209 would like to keep it open because she has linked it to the blog.